docs/ReleaseNotes.md

```diff
@@ -0,0 +1,45 @@
+# Release Notes
+This document contains information about the releases of this crate.
+
+## Version 0.1.0
+The Networking team is happy to announce the release of 0.1.0 of the [laminar crate](https://github.com/amethyst/laminar). 
+It provides UDP networking modified for the needs of game networking. 
+Most of the techniques used were published and detailed by [Glenn Fiedler](https://gafferongames.com/). 
+We’d like to extend a special thanks to him and his articles.
+
+**Features**
+- UDP-based protocol
+- Automatic Fragmentation
+- RTT estimation
+- Connection tracking.
+- Unreliable and Reliable sending of packets.
+- Protocol version monitoring.
+- A link conditioner to simulate packet loss and latency
+- Good error handling with **zero** panics.
+- Well tested by integration and unit tests
+- Benchmarks
+
+**Workflow**
+
+As a team, we adopted the following the workflow agreements. When we begin work on the amethyst_network crate, we’ll use the same agreements. They are focused on maintaining a high level of quality in the code, and for working with a highly distributed team. We’re including them here as some of the other teams may find them of use.
+
+- All warnings produced by `cargo test` are treated as errors by the CI/CD system
+- All 'Clippy' warnings are treated as errors by the CI/CD system
```

Comment:
These next two lines are currently not applicable to `amethyst_network` because `amethyst` doesn't use them. We can separate them out just for `laminar`.

cc @fhaynes

Reply:
so for the moment its fine but remove the part about `amethyst_network` in reality this stuff related to `amethyst_network` should live within that crate.